technic/init.lua

```diff
@@ -43,15 +44,18 @@ dofile(modpath.."/max_lag.lua")
 -- Helper functions
 dofile(modpath.."/helpers.lua")
 
+-- Register functions
+dofile(modpath.."/register.lua")
+
+-- Compatibility shims for tools
+dofile(modpath.."/machines/compat/tools.lua")
```

Comment:
I think it would be better to have the `compat` directory in the `technic` directory, not under `machines`.

Reply:
I do agree with that and already moved it once few days ago...
But then moved it back after going through local merge between branches for testing because it would divide stuff with digtron and cause unnecessary conflicts with #230 so decided to keep all compatibility things in same place for now.

Reply:
Let these be for now and reorder after this PR (and maybe also #230) is merged and table is clean. Need separate issue?